### DISCUSSION TITLE
Feature Request: Show Optional Reviewers in PR

### DISCUSSION THREAD

**Discussion by meiram-tr** [community/community#116796] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

# Feature Request: Show Optional Reviewers in PR

## Description
Currently, when a CODEOWNERS file lists multiple developers for a specific directory or file of code, all of them are automatically added as required reviewers when a pull request is opened. This can sometimes lead to unnecessary confusion and delays in the review process, especially when only one approval is required.

I propose introducing a feature that add a new mark specifying optional reviewers after one of them approve the PR . Optional reviewers would still be notified of the pull request but their approval wouldn't be required for the pull request to be merged. This would streamline the review process and increase the clarity of non-required reviewers and required reviewers.

## Example
Consider the following scenario in a CODEOWNERS file:
```text
# CODEOWNERS

# User F is responsible for the backend directory
/backend/ @user-f

# User A, User B and User C are responsible for section-a and others owners different sections
/frontend/section-a/ @user-a @user-b @user-c
/frontend/section-b/ @user-d @user-f
```

If a pr contains changes in "/frontend/section-a/"
@team-b & @team-c will be added automatically to the reviewers list

Once @team-b approve the PR the PR become pass the codeowners restriction while the visual list show like we are missing approval from @team-c

## Additional Information
This feature would improve the efficiency of the review process, particularly in large teams where multiple developers might be responsible for same parts of the codebase.

## Implementation Suggestion
- Add an optional design to reviewers list.
- In this suggestion i changed the circle to be green:
<img width="326" alt="image" src="https://github.com/community/community/assets/7975804/de534ea5-97c3-40d6-b1a7-93f8ac850824">

Thank you for considering this feature request.

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @meiram-tr!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Pull Requests category. I've gone ahead and moved it for you. Good luck!

**Comment by meiram-tr** [comment]
Thank you, appreciate your help!
